python/delta_sharing/rest_client.py

```diff
@@ -167,6 +172,11 @@ def _get_internal(self, target: str, data: Optional[Dict[str, Any]] = None):
     def _post_internal(self, target: str, data: Optional[Dict[str, Any]] = None):
         return self._request_internal(request=self._session.post, target=target, data=data)
 
+    def _request_internal_with_backoff(self, request, target: str, data: Optional[Dict[str, Any]]):
```

Comment:
Who will call this? Or is that out of scope of this PR?

Reply:
Oops I forgot to change `_get_internal` and `_post_internal` to call this